OMMatlab.m

```diff
@@ -781,7 +781,7 @@ function simulate(obj,resultfile,simflags)
             if exist('lintime', 'var')
                 linruntime=join([getexefile, ' -l=', char(lintime)]);
             else
-                linruntime=join([getexefile, ' -l=', obj.linearOptions.('stopTime')]);
+                linruntime=join([getexefile, strcat(' -l=',char(obj.linearOptions.('stopTime')))]);

```

Comment:
why r u using strcat ? I guess it should be ` join([getexefile, ' -l=', char(obj.linearOptions.('stopTime'))])`